picard/ui/matchdetailsdialog/__init__.py

```diff
@@ -0,0 +1,278 @@
+        else:
+            self.acoustid_lookup_url.hide()
+
+    def set_input_filename_lbl(self, p_filename):
+        self.input_filename_lbl.setText("Input File: " + p_filename)
+
+    def get_value_for_row_id(self, row, value):
+        return value
+
+    def setupUi(self):
+        self.verticalLayout = QtWidgets.QVBoxLayout(self)
+        self.verticalLayout.setObjectName("vertical_layout")
+
+        self.input_filename_widget = QtWidgets.QWidget(self)
+        self.input_filename_lbl = QtWidgets.QLabel(self.input_filename_widget)
+        self.input_filename_lbl.setFocusPolicy(QtCore.Qt.StrongFocus)
+        myFont = QtGui.QFont()
+        myFont.setBold(True)
+        self.input_filename_lbl.setFont(myFont)
+        self.input_filename_lbl.setTextInteractionFlags(QtCore.Qt.TextSelectableByMouse)
+        self.verticalLayout.addWidget(self.input_filename_lbl)
+
+        self.tip_widget = QtWidgets.QWidget(self)
+        self.tip_lbl = QtWidgets.QLabel(self.tip_widget)
+        self.tip_lbl.setText(_("Tip: Column header left click to sort, right click to filter"))
+        self.tip_lbl.setMargin(10)
+        self.verticalLayout.addWidget(self.tip_lbl)
+
+        self.center_widget = QtWidgets.QWidget(self)
+        self.center_widget.setObjectName("center_widget")
+        self.center_layout = QtWidgets.QVBoxLayout(self.center_widget)
+        self.center_layout.setObjectName("center_layout")
+        self.center_layout.setContentsMargins(1, 1, 1, 1)
+        self.center_widget.setLayout(self.center_layout)
+        self.verticalLayout.addWidget(self.center_widget)
+
+        self.acoustid_lookup_url_widget = QtWidgets.QWidget(self)
+        self.acoustid_lookup_url = QtWidgets.QLabel(self.acoustid_lookup_url_widget)
+        self.acoustid_lookup_url.setOpenExternalLinks(True)
+        self.verticalLayout.addWidget(self.acoustid_lookup_url)
+
+        self.buttonBox = QtWidgets.QDialogButtonBox(self)
+        self.export_button = QtWidgets.QPushButton("Export Results", self.buttonBox)
+        self.export_button.setEnabled(True)
+        self.buttonBox.addButton(self.export_button, QtWidgets.QDialogButtonBox.ActionRole)
+        self.export_button.clicked.connect(self.export_data)
+
+        self.accept_button = QtWidgets.QPushButton("Load into Picard", self.buttonBox)
+        self.accept_button.setEnabled(False)
+        self.buttonBox.addButton(self.accept_button, QtWidgets.QDialogButtonBox.AcceptRole)
+
+        self.buttonBox.addButton(StandardButton(StandardButton.CANCEL), QtWidgets.QDialogButtonBox.RejectRole)
+        self.buttonBox.accepted.connect(self.accept)
+        self.buttonBox.rejected.connect(self.reject)
+        self.verticalLayout.addWidget(self.buttonBox)
+
+    # Override
+    def show_table(self, selected_row_nbr=-1, sort_column=None, sort_order=QtCore.Qt.DescendingOrder):
```

Comment:
You override with a different prototype, Codacy will complain.

Perhaps define overriden method as:
```python
def show_table(self, sort_column=None, sort_order=QtCore.Qt.DescendingOrder, **kwargs):
```

and this one as:

```python
def show_table(self, sort_column=None, sort_order=QtCore.Qt.DescendingOrder, selected_row_nbr=-1, **kwargs):
```

Not sure.